### ISSUE TITLE
Cannot parse a non JSON Value from local storage

### ISSUE THREAD

**Issue by ElMatella** [bevacqua/local-storage#20]
Hi,

I recently had an issue where the localstorage system was corrupted, and so the `ls.get()` function returned an error:

Reproduce with this local storage content::

```
|-----|-----------|
| foo | undefined |
|-----|-----------|
```

```
ls.get('foo') // -> Throws an error because undefined is not a valid JSON.
```

Did this ever happened to one of you? How did you manage the thing?

For the time being, I use a `try/catch` and delete the key of the localstorage if such a thing happens.

**Comment by maaso** [comment]
I'm seeing the same behaviour, thanks for the`try/catch` tip :)

**Comment by jasonbarry** [comment]
I've encountered this too — it makes it difficult to easily migrate from `window.localStorage`

**Comment by bevacqua** [comment]
Fixed in `2.0.0`
